canonical-websites/www.ubuntu.com
# Update the openstack contact-us and thank-you pages

## Done

- updated the training [copy doc](docs.google.com/document/d/1gyxy75qXC5Ta08o6Vnm13yK6aeN4qOnsETB2gZ4e0ZY/edit#)
- updated the contact-us [copy doc](docs.google.com/document/d/10lo3O2TKiR-cc48CqhH3Ghak0xZQaP2n4YthDtdIXpo/edit#)
- updated the thank-you [copy doc](docs.google.com/document/d/1-dWhd0ydDWlYg16rEybPPmkQ6cdxsBmiFOuUz0ApCBM/edit#heading=h.qn5fq8d3phy8)
- updated the storage [copy doc](https://docs.google.com/document/d/1iw2SWNmoj4GZ2pUh0yB8gbtVXK0oXfyEaXT44bQbPlI/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at:
  - http://0.0.0.0:8001/openstack/contact-us?product=openstack-training
  - http://0.0.0.0:8001/openstack/contact-us?product=openstack-training-classroom
  - http://0.0.0.0:8001/openstack/contact-us?product=openstack-training-onsite
  - http://0.0.0.0:8001/openstack/contact-us?product=openstack-training-server-admin
  - http://0.0.0.0:8001/openstack/contact-us?product=ubuntu-advantage-storage
  - http://0.0.0.0:8001/openstack/contact-us?product=openstack-managed-cloud
  - http://0.0.0.0:8001/openstack/contact-us?product=openstack-managed-cloud-demo
  - http://0.0.0.0:8001/openstack/contact-us?product=foundation-cloud
  - http://0.0.0.0:8001/openstack/contact-us?product=openstack
  - http://0.0.0.0:8001/openstack/contact-us
  - http://0.0.0.0:8001/openstack/thank-you?product=openstack-training
  - http://0.0.0.0:8001/openstack/thank-you?product=openstack-training-classroom
  - http://0.0.0.0:8001/openstack/thank-you?product=openstack-training-onsite
  - http://0.0.0.0:8001/openstack/thank-you?product=openstack-training-server-admin
  - http://0.0.0.0:8001/openstack/thank-you?product=ubuntu-advantage-storage
  - http://0.0.0.0:8001/openstack/thank-you?product=openstack-managed-cloud
  - http://0.0.0.0:8001/openstack/thank-you?product=openstack-managed-cloud-demo
  - http://0.0.0.0:8001/openstack/thank-you?product=foundation-cloud
  - http://0.0.0.0:8001/openstack/thank-you

## Issue / Card

Fixes #3364, Fixes #3353 

## Screenshots

[if relevant, include a screenshot]
